pipelines/slim/minimal_feature.model.css

```
@@ -6,17 +6,17 @@
- * The `model_name` honors `DARK_FACTORY_PLAN_MODEL` via `${VAR:-default}`
+ * The `model_name` honors `DARK_FACTORY_MINIMAX_MODEL` via `${VAR:-default}`
  * env-var substitution (see runner/parser.py:_substitute_env_in_value).
  *
  * Review tier: independent adversarial backend (Antigravity), separate from
  * the coder --backend.
  */
 .plan {
-  backend: claude;
-  model_name: ${DARK_FACTORY_PLAN_MODEL:-claude-opus-4-6};
+  backend: minimax;
+  model_name: ${DARK_FACTORY_MINIMAX_MODEL:-MiniMax-M3};
```

Stylelint (17.14.0)

[error] Invalid position for declaration (no-invalid-position-declaration)

(no-invalid-position-declaration)